Documentation/ceph-upgrade.md

```diff
@@ -259,10 +259,31 @@ At this point, your Rook operator should be running version `rook/ceph:v1.0.0`
 
 Verify the Ceph cluster's health using the [health verification section](#health-verification).
 
+### 6. Update the Mon Ports
+
+The port on which the mons are listening has changed in the 1.0 release. Previously, the mons were listening on
```

Comment:
This change likely seems like an arbitrary annoyance to the reader unless we explain why this decision was made.

Reply:
sure, will give more background